package NPrsLog.UniformBoundCompressionProto;

message TSingleStatCompressionMeta {
    required float MinValue = 1;
    optional float MaxValue = 2;
}

message TCompressedFactor {
    required TSingleStatCompressionMeta Meta = 1;
    optional string CompressedFeatures = 2;
}

message TCompressedWebData {
    repeated string DocIds = 1;
    repeated string Urls = 2;
    optional bytes WebFeaturesIds = 3;
    optional bytes WebMetaFeaturesIds = 4;
    optional bytes RapidClicksFeaturesIds = 10;
    repeated TCompressedFactor Statistics = 5;
    repeated TCompressedFactor WebFactors = 6;
    repeated TCompressedFactor WebMetaFactors = 7;
    repeated TCompressedFactor RapidClicksFactors = 11;
    repeated bytes OmniTitles = 8;
    repeated bytes SerializedFactors = 9;
}
